packages/utils/assert-never/README.md

```diff
@@ -0,0 +1,25 @@
+# AssertNever
+
+This package contains a utility function which helps ensure that all entries of an enum are handled in a switch case statement.
+
+## Example
+
+```ts
+enum A {
+  foo,
+  bar,
+}
+
+...
```

Comment:
Could we also add an example with a union type since that's more common in typescript / gives us the built in type checking? Something like?

```
type Size = 'small' | 'medium' | 'large' 

const ringSize: Size = ...

 switch(ringSize) {
   case 'small':      
    return 'small';
   case 'medium':
     return 'medium';
   case 'large':
     return 'large';
   default:
     assertNever(ringSize); // Will cause build error if another entry is added to Size
 }

```

Reply:
Sure, can do.